### PR TITLE
Improve VoiceControl support

### PR DESCRIPTION
**Pull request by jacquealvesb**
## 📝 Docs

- [x] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?

## 📲 What

Improvement of VoiceControl support

## 🤔 Why

Issue #263

### CODE REVIEW

**Comment by mltbnz** [comment]
Thanks for your PR.
I am very sad that the Greetings step in the CI failed because it should have said something like: "Hello and thanks for your contribution"... Need to check that.

Regarding your PR:
I did not know `accessibilityUserInputLabels` and first had to check it out. Is it the case that it adds additional description to a e.g. label and are they used in addition to `accessibiltyLabel`?

Regarding the lokalizable changes: I think they are good but we started using a service to manage: https://crowdin.com/project/critical-maps Maybe you could also update them there? It is not automated yet and I manually have to download the strings files and add them to the repo. Maybe as a contributor that also works

File: CriticalMass/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -3,7 +3,7 @@
 "chat.noChatActivity" = "Niemand chattet derzeit...\nWarum fängt du nicht an?";
 
 /* No comment provided by engineer. */
-"chat.placeholder" = "Nachricht ...";
+"chat.placeholder" = "Nachricht Schreiben";
```
Comment (mltbnz) [inline CriticalMass/de.lproj/Localizable.strings:6]: Think this needs to be `Nachtricht schreiben`

File: CriticalMass/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -150,7 +153,7 @@
 "settings.openSource" = "GitHub";
 
 /* No comment provided by engineer. */
-"settings.opensource.action" = "Projekt ansehen";
+"settings.opensource.action" = "Repository Ansehen";
```
Comment (mltbnz) [inline CriticalMass/de.lproj/Localizable.strings:156]: -> "Repository ansehen"

File: CriticalMass/de.lproj/Localizable.strings (hunk 1 above)
Comment (fbernutz) [reply CriticalMass/de.lproj/Localizable.strings:6]:
```suggestion
"chat.placeholder" = "Nachricht schreiben";
```

It should just be a small `s` :)

File: CriticalMass/de.lproj/Localizable.strings (hunk 2 above)
Comment (fbernutz) [reply CriticalMass/de.lproj/Localizable.strings:156]:
```suggestion
"settings.opensource.action" = "Repository ansehen";
```

**Comment by fbernutz** [comment]
Thanks again for the contribution! 
Could you please resolve the conversations (you can also commit the suggestions) and rebase your branch with master? Then we could merge this 🎉

**Comment by jacquealvesb** [comment]
> Thanks for your PR.
> I am very sad that the Greetings step in the CI failed because it should have said something like: "Hello and thanks for your contribution"... Need to check that.
> 
> Regarding your PR:
> I did not know `accessibilityUserInputLabels` and first had to check it out. Is it the case that it adds additional description to a e.g. label and are they used in addition to `accessibiltyLabel`?
> 
> Regarding the lokalizable changes: I think they are good but we started using a service to manage: https://crowdin.com/project/critical-maps Maybe you could also update them there? It is not automated yet and I manually have to download the strings files and add them to the repo. Maybe as a contributor that also works

Hi, Malte

As far as I know, the `accessibilityUserInputLabels` are only used in VoiceControl, when someone says any of the labels on the array the VoiceControl assumes it is about that element the person is talking about, `accessibiltyLabel` is used on VoiceOver.

Also, I'll try to update the lokalizable changes on crowdin. 😊

**Review by fbernutz — APPROVED** [review]
looks good to me, @mltbnz do you have anything to add or can we merge this?

**Pull request merged**